reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/problemsummary/GetProblemSummariesMethod.java

```diff
@@ -99,6 +112,10 @@ public int compare(String severity1, String severity2)
         ClassificationService classificationService = new ClassificationService(context);
         for (ClassificationModel classification : classificationService.findAll())
         {
+            Set<String> tags = classification.getTags();
+            if (!TagUtil.isTagsMatch(tags, includeTags, excludeTags, false))
```

Comment:
well, I think it will be pretty hard to come up with a method name because it does a lot of things :). Maybe we can have some kind of TagValidator that may set up several things and then run "validate()". With this we are able to include some new options in the future (in case it even makes sense)